tests/swarm-test.js

```diff
@@ -38,9 +38,16 @@ beforeEach(function (done) {
 })
 
 afterEach(function (done) {
-  swarmA.closeListener()
-  swarmB.closeListener()
-  done()
+  // This should be 2, but for some reason
+  // that will fail in most of the tests
```

Comment:
@diasdavid not sure what is going wrong here, but we need to wait for all things to be closed to start the next test, otherwise I get race conditions and ports in use errors.